src/Microsoft.Framework.NullLogging.Internal/Microsoft.Framework.NullLogging.Internal.kproj

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Run `\\GarnetA\Share\KProjUpdateScript\UpdateKProjectFiles.ps1` on this (can just run it in global.json directory)